leptos_reactive/src/trigger.rs

```diff
@@ -18,6 +18,47 @@ pub struct Trigger {
 }
 
 impl Trigger {
+    /// Creates a [`Trigger`](crate::Trigger), a kind of reactive primitive.
+    ///
+    /// A trigger is a data-less signal with the sole purpose
+    /// of notifying other reactive code of a change. This can be useful
+    /// for when using external data not stored in signals, for example.
+    ///
+    /// This is identical to [`create_rw_signal`].
```

Comment:
should read "identical to [\`create_trigger\`]" right?

Reply:
Ah sorry good catch, it should

Reply:
All fixed!